packages/react-component-library/src/components/List/ListItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import { IconChevronRight } from '@royalnavy/icon-library'
+
+export interface ListItemProps extends ComponentWithClass {
+  children: string
+  isActive?: boolean
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onMouseEnter?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onMouseLeave?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  title: string
+}
+
+export const ListItem: React.FC<ListItemProps> = ({
+  children,
+  isActive,
+  className,
+  onClick,
+  onMouseEnter,
+  onMouseLeave,
+  title,
+}) => {
+  const classes = classNames(
+    'rn-list__item',
+    {
+      'is-inactive': isActive === false,
+    },
+    className
+  )
+
+  return (
+    <li className={classes} data-testid="list-item">
+      <button
+        className="rn-list__item-content"
+        onClick={onClick}
+        onMouseEnter={onMouseEnter}
+        onMouseLeave={onMouseLeave}
+      >
+        <div>
+          <h2 className="rn-list__item-title">{title}</h2>
```

Comment:
Is `h2` the right tag? Maybe not a big deal - open to debate I think.

Reply:
Having spoken with @hxltrhuxze we feel `h2` is fine at this stage.